x-pack/plugins/index_lifecycle_management/public/plugin.tsx

```diff
@@ -110,7 +110,11 @@ export class IndexLifecycleManagementPlugin
         addAllExtensions(indexManagement.extensionsService);
       }
 
-      registerUrlGenerator(coreSetup, management, share);
+      plugins.share.url.locators.create(
```

Comment:
shouldn't we register this on the server as well ?

Reply:
Ideally we would also have it on the server, too. But ILM URL generator uses internally few functions that are available only on the client, so I did not port it to the server. I leave it for the Stack Management team to port it to the server if they want to.